frame/vault/src/lib.rs

```diff
@@ -105,6 +104,7 @@ pub mod pallet {
 		ArithmeticError, DispatchError, FixedPointNumber, Perquintill,
 	};
 	use sp_std::fmt::Debug;
+	use std::cmp::Ordering;
```

Comment:
AFAIK we can't use `std` library from Rust, only `sp-std` from Substrate.

P.S. Before committing try to use `cargo check --all-targets --all-features --workspace` command for self-checking.

Reply:
>   error[E0433]: failed to resolve: use of undeclared crate or module `std`